js/src/cli/namespace.ts

```diff
@@ -145,4 +171,45 @@ programCommand("show_namespace")
     }
   });
 
+programCommand("update_namespace")
+  .requiredOption(
+    "-cp, --config-path <string>",
+    "JSON file with mint, namespaceToken, and updates to one or many of: prettyName, permissivenessSettings, or whitelistedStakingMints"
+  )
+  .action(async (_files: string[], cmd) => {
+    log.setLevel("info");
+    const { keypair, env, rpcUrl, configPath } = cmd.opts();
+
+    const walletKeyPair = loadWalletKey(keypair);
+    const anchorProgram = await getNamespaceProgram(walletKeyPair, env, rpcUrl);
+
+    if (configPath === undefined) {
```

Comment:
Does commander not throw any exceptions before this since it's a `requiredOption`? I realize there are other examples of this in the codebase.

Reply:
I think it'll throw